expected/12.16/extra/bitstring.out

```diff
@@ -0,0 +1,810 @@
+--SET log_min_messages  TO DEBUG1;
```

Comment:
Is this come from PR #95?

Reply:
Yes, if here was big test changes around of conversion error `CONTEXT` there is reason make all formal test normalisation here.

Reply:
Is this test just moved from type.sql without any modify?
If so, there is no problem. Please help me resolve this thread.

Reply:
Yes, @nxhai98 , this is just moved test without any TC modifying. Resolved.